Source/CarthageKit/Git.swift

```diff
@@ -524,21 +571,33 @@ public func resolveReferenceInRepository(_ repositoryFileURL: URL, _ reference:
 	return ensureDirectoryExistsAtURL(repositoryFileURL)
 		.then(launchGitTask([ "rev-parse", "\(reference)^{object}" ], repositoryFileURL: repositoryFileURL))
 		.map { string in string.trimmingCharacters(in: .whitespacesAndNewlines) }
-		.mapError { error in CarthageError.repositoryCheckoutFailed(workingDirectoryURL: repositoryFileURL, reason: "No object named \"\(reference)\" exists", underlyingError: error as NSError) }
+		.mapError { error in
+			CarthageError.repositoryCheckoutFailed(
+				workingDirectoryURL: repositoryFileURL,
+				reason: "No object named \"\(reference)\" exists",
+				underlyingError: error as NSError
+			)
+		}
 }
 
 /// Attempts to resolve the given tag into an object SHA.
 internal func resolveTagInRepository(_ repositoryFileURL: URL, _ tag: String) -> SignalProducer<String, CarthageError> {
 	return launchGitTask([ "show-ref", "--tags", "--hash", tag ], repositoryFileURL: repositoryFileURL)
 		.map { string in string.trimmingCharacters(in: .whitespacesAndNewlines) }
-		.mapError { error in CarthageError.repositoryCheckoutFailed(workingDirectoryURL: repositoryFileURL, reason: "No tag named \"\(tag)\" exists", underlyingError: error as NSError) }
+		.mapError { error in
+			CarthageError.repositoryCheckoutFailed(
+				workingDirectoryURL: repositoryFileURL,
+				reason: "No tag named \"\(tag)\" exists",
+				underlyingError: error as NSError
+			)
+		}
 }
 
 /// Attempts to determine whether the given directory represents a Git
 /// repository.
 public func isGitRepository(_ directoryURL: URL) -> SignalProducer<Bool, NoError> {
 	return ensureDirectoryExistsAtURL(directoryURL)
-		.then(launchGitTask([ "rev-parse", "--git-dir", ], repositoryFileURL: directoryURL))
+		.then(launchGitTask([ "rev-parse", "--git-dir" ], repositoryFileURL: directoryURL))
```

Comment:
The approach of the SwiftLint rule is to only require a trailing comma on multi-line arrays, not on single line ones like here. Also I'd like to bring up the leading and trailing whitespace in arrays like in `[ ` and ` ]`. I never put whitespaces there, you always seem to do. Is this a rule, too, to have a single whitespace in single-line arrays? If so, I'd like to automate this as well.